modules/update-mirror-repository.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="update-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Before you upgrade a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+.Procedure
+
+. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions.
+
+. Set the required environment variables:
++
+----
+$ OCP_RELEASE=<release_version> # <1>
+$ LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' # <2>
+$ LOCAL_REPOSITORY='<local_repository_name>' # <3>
+$ PRODUCT_REPO='openshift-release-dev' # <4>
+$ LOCAL_SECRET_JSON='<path_to_pull_secret>' # <5>
+$ RELEASE_NAME='ocp-release' # <6>
+$ ARCHITECTURE=<server_architecture> # <7>
+$ REMOVABLE_MEDIA_PATH=<path> <8>
+----
+<1> For `<release_version>`, specify the tag that corresponds to the version of {product-title} to which you want to upgrade, such as `4.5.0`.
+<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror
+repository, and for `<local_registry_host_port>`, specify the port that it
+serves content on.
+<3> For `<repository_name>`, specify the name of the repository to create in your
+registry, such as `ocp4/openshift4`.
+<4> The repository to mirror. For a production release, you must specify
+`openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
+the pull secret for your mirror registry that you created.
+<6> For a production release, you must specify
+`ocp-release`.
+<7> For `<server_architecture>`, specify the architecture of the server, such as `x86_64`.
+<8> For `<path>`, specify the path to the directory to host the mirrored images.
+
+. Review the images and configuration manifests to mirror:
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
+----
+. Mirror the version images to the internal container registry:
+.. If your mirror host does not have internet access, connect the removable media to a system that is connected to the internet.
```

Comment:
On the other hand, here we removed `If the local container registry and the cluster are connected to the mirror host, directly push the release images to the local registry` scenario, right? I think this scenario is still valid, just not support `--apply-release-image-signature` option, that means user can run `oc adm release mirror -a ${LOCAL_SECRET_JSON} --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}`, then need to create an image signature ConfigMap manually.

Reply:
@jianlinliu, since https://github.com/openshift/openshift-docs/pull/24971 is going to address the ConfigMap, is this PR ready to merge?

Reply:
Yes.